app/views/profiles/show.html.erb

```diff
@@ -1,7 +1,8 @@
 <div class="p-4">
   <div class="grid gap-4 xl:grid-cols-2 2xl:grid-cols-3">
     <div class="p-4 bg-white border border-gray-200 rounded-lg shadow-sm 2xl:col-span-2 dark:border-gray-700 sm:p-6 dark:bg-gray-800">
```

Comment:
**suggestion (non-blocking):** Since you are updating this file, can you make this small change:

Change line 3 to be: `<%= viral_card do %>`
Update closing div on line 6 to be `<% end %>`

Reply:
Fixed.